README.md
# mgt
mgt


history
   33  systemctl restart php7.4-fpm nginx
   34  cd /var/www/html/magento2/
   35  ls
   36  bin/magento sampledata:deploy
   37  vim composer.json 
   38  php bin/magento sampledata:deploy --no-update
   39  php bin/magento sampledata:remove -h
   40  php bin/magento sampledata:remove 
   41  php bin/magento sampledata:deploy 
   42  php bin/magento setup:upgrade
   43  systemctl start elasticsearch
   44  systemctl enable elasticsearch
   45  php bin/magento setup:upgrade
   46  php bin/magento sampledata:deploy 
   47  composer update
   48  php bin/magento sampledata:deploy 
   49  php bin/magento setup:upgrade
   50  free -m
   51  systemctl start elasticsearch
   52  php bin/magento setup:upgrade
   53  php bin/magento sampledata:deploy 
   54  composer version
   55  composer help
   56  composer -v
   57  composer about
   58  composer info
   59  composer -v
   60  compser upgrade
   61  composer upgrade
   62  df -hT
   63  history
   64  composer upgrade
   65  sudo composer self-update
   66  sudo composer self-update --2
   67  composer upgrade
   68  apt install php7.4*
   69  apt install php7.4-gd php7.4-bcmath php7.4-cgi php7.4-cli php7.4-opcache php7.4-common php7.4-ast php7.4-iconv php7.4-sybase php7.4-gmagik
   70  apt install php7.4-gd php7.4-bcmath php7.4-cgi php7.4-cli php7.4-opcache php7.4-common php7.4-ast php7.4-iconv php7.4-sybase 
   71  apt install php7.4-bcmath php7.4-tideways php7.4-mailparse php7.4-soap* php7.4-json php7.4-lua* php7.4-xml* php7.4-http php7.4-imagik php7.4-stomp* php7.4-memcached php7.4-imap*
   72  apt install php7.4-bcmath php7.4-tideways php7.4-mailparse php7.4-soap* php7.4-json php7.4-lua* php7.4-xml* php7.4-http php7.4-stomp* php7.4-memcached php7.4-imap*
   73  apt install php7.4-snmp php7.4-geoip php7.4-tidey php7.4-dom php7.4-curl php7.4-json*
   74  apt install php7.4-snmp php7.4-geoip  php7.4-dom php7.4-curl php7.4-json*
   75  systemctl restart php7.4-fpm.service 
   76  sudo composer self-update --2
   77  composer upgrade
   78  apt install php7.4-bcmath php7.4-ctype php7.4-curl php7.4-dom php7.4-gd php7.4-hash php7.4-iconv php7.4-intl php7.4-mbstring php7.4-openssl php7.4-pdo_mysqlphp7.4-simplexml
   79  apt install php7.4-bcmath php7.4-ctype php7.4-curl php7.4-dom php7.4-gd php7.4-hash php7.4-iconv php7.4-intl php7.4-mbstring php7.4-openssl php7.4-pdo_mysql php7.4-simplexml
   80  apt install php7.4-bcmath php7.4-ctype php7.4-curl php7.4-dom php7.4-gd  php7.4-iconv php7.4-intl php7.4-mbstring 
   81  apt autoremove 
   82  php7.4-soap php7.4-xsl php7.4-zip php7.4-sockets
   83  apt install php7.4-soap php7.4-xsl php7.4-zip php7.4-sockets
   84  systemctl restart php7.4-fpm.service 
   85  composer upgrade
   86  apt install php7.4-soap php7.4-xsl php7.4-zip php7.4-raphf'
   87  ;
   88  apt install php7.4-soap php7.4-xsl php7.4-zip php7.4-raphf
   89  systemctl restart php7.4-fpm.service 
   90  composer upgrade
   91  apt install php7.4-soap php7.4-xsl php7.4-zip php7.4-propro
   92  systemctl restart php7.4-fpm.service 
   93  composer upgrade
   94  composer selfupdate --1
   95  composer upgrade
   96  df -hT
   97  halt
   98  mkdir hdd
   99  mkdir /hdd
  100  swapoff /swapfile 
  101  rm -rf /swapfile 
  102  rm -rf /swapfile 
  103  swapoff /swapfile 
  104  swapoff -f /swapfile 
  105  lsblk
  106  mkswap /dev/sdb
  107  vim /etc/fstab 
  108  init 6
  109  free -m
  110  rm -rf /swapfile 
  111  df -hT
  112  cd /var/www/html/magento2/
  113  ls
  114  composer upgrade
  115  composer upgrade
  116  bin/magento sampledata:deploy
  117  Copy
  118  bin/magento setup:upgrade
  119  composer selfupdate --2
  120  composer upgrade
  121  bin/magento setup:upgrade
  122  Copy
  123  Updated
  124  9 days ago
  125  bin/magento sampledata:deploy
  126  composer upgrade
  127   composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition <install-directory-name>
  128  cd ..
  129  mkdir magento/
  130  cp -arv magento2/* magento/
  131  rm -rf magento2/*
  132  cp -arv magento/phpmyadmin magento2/
  133  ls
  134  cd magento2/
  135   composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition <install-directory-name>
  136   composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition /var/www/html/magento2
  137   composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition /var/www/html/magento
  138   composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition /var/
  139   composer create-project --repository-url=https://repo.magento.com/magento/project-community-edition /var/www/html/magento2
  140  composer create-project --repository-url=https://repo.magento.com/ magento/project-enterprise-edition=2.3.0 /var/www/html/magento2
  141  git clone https://github.com/magento/magento2.git
  142  composer upgrade
  143  git checkout 2.4.1
  144  composer install
  145  mv magento2/* /var/www/html/magento2/
  146  composer upgrade
  147  git checkout 2.4.1
  148  composer upgrade
  149  sudo composer self-update --1
  150  composer upgrade
  151  php bin/magento setup:install --base-url=https://magento2.mgt/ --db-host=localhost --db-name=magento --db-user=user --db-password=password  --admin-firstname=MGT --admin-lastname=COM --admin-email=test@mgt.com  --admin-user=admin --admin-password=Gopal@123 --language=en_US  --currency=INR --timezone=Asia/Kolkata --use-rewrites=1 --search-engine=elasticsearch7 --elasticsearch-host=http://127.0.0.1/ --elasticsearch-port=9200
  152  systemctl start elasticsearch

env.php
<?php
return [
    'backend' => [
        'frontName' => 'mgtAdmin'
    ],
    'queue' => [
        'consumers_wait_for_messages' => 1
    ],
    'crypt' => [
        'key' => '18Av6ITivOZG3gwY1DhMDWtlLfx1spLP'
    ],
    'db' => [
        'table_prefix' => '',
        'connection' => [
            'default' => [
                'host' => 'localhost',
                'dbname' => 'magento3',
                'username' => 'magento2',
                'password' => 'magento2',
                'model' => 'mysql4',
                'engine' => 'innodb',
                'initStatements' => 'SET NAMES utf8;',
                'active' => '1',
                'driver_options' => [
                    1014 => false
                ]
            ]
        ]
    ],
    'resource' => [
        'default_setup' => [
            'connection' => 'default'
        ]
    ],
    'x-frame-options' => 'SAMEORIGIN',
    'MAGE_MODE' => 'developer',
    'session' => [
        'save' => 'redis',
        'redis' => [
            'host' => '127.0.0.1',
            'port' => '6379',
            'password' => '',
            'timeout' => '2.5',
            'persistent_identifier' => '',
            'database' => '2',
            'compression_threshold' => '2048',
            'compression_library' => 'gzip',
            'log_level' => '4',
            'max_concurrency' => '6',
            'break_after_frontend' => '5',
            'break_after_adminhtml' => '30',
            'first_lifetime' => '600',
            'bot_first_lifetime' => '60',
            'bot_lifetime' => '7200',
            'disable_locking' => '0',
            'min_lifetime' => '60',
            'max_lifetime' => '2592000',
            'sentinel_master' => '',
            'sentinel_servers' => '',
            'sentinel_connect_retries' => '5',
            'sentinel_verify_master' => '0'
        ]
    ],
    'cache' => [
        'frontend' => [
            'default' => [
                'id_prefix' => '1d7_'
            ],
            'page_cache' => [
                'id_prefix' => '1d7_',
                'backend' => 'Magento\\Framework\\Cache\\Backend\\Redis',
                'backend_options' => [
                    'server' => '127.0.0.1',
                    'database' => '3',
                    'port' => '6379',
                    'password' => '',
                    'compress_data' => '0',
                    'compression_lib' => ''
                ]
            ]
        ],
        'allow_parallel_generation' => false
    ],
    'lock' => [
        'provider' => 'db',
        'config' => [
            'prefix' => ''
        ]
    ],
    'directories' => [
        'document_root_is_pub' => false
    ],
    'cache_types' => [
        'config' => 1,
        'layout' => 1,
        'block_html' => 1,
        'collections' => 1,
        'reflection' => 1,
        'db_ddl' => 1,
        'compiled_config' => 1,
        'eav' => 1,
        'customer_notification' => 1,
        'config_integration' => 1,
        'config_integration_api' => 1,
        'full_page' => 1,
        'config_webservice' => 1,
        'translate' => 1,
        'vertex' => 1
    ],
    'downloadable_domains' => [
        'magento.mgt'
    ],
    'install' => [
        'date' => 'Mon, 08 Feb 2021 13:36:13 +0000'
    ]
];
